providers/aws/resources/aws.lr

```diff
@@ -2553,6 +2555,27 @@ private aws.config.recorder @defaults("name region") {
   resourceTypes []string
 }
 
+// AWS config delivery channel
+private aws.config.deliverychannel @defaults("name region") {
+  // Name of the delivery channel
+  name string
+  // S3 bucket name where configuration snapshots are delivered
+  s3BucketName string
+  // Prefix for the S3 bucket where configuration snapshots are delivered
+  s3KeyPrefix string
+  // ARN of the SNS topic that AWS Config delivers notifications to
+  snsTopicARN string
+  // Region for the delivery channel
+  region string
+}
+
+
```

Comment:
@HRouhani Can you delete the empty lines here?

Reply:
sure, no worries